ts/models/conversation.ts

```diff
@@ -705,6 +731,37 @@ export class ConversationModel extends Backbone.Model<ConversationAttributes> {
     }
   }
 
+  public async sendMessageRequestResponse(isApproved: boolean) {
+    const publicKey = fromHexToArray(getOurPubKeyStrFromCache());
+    // const msgRequestResponseParams:
+    const timestamp = Date.now();
+
+    const messageRequestResponseParams = {
+      timestamp,
+      publicKey,
+      isApproved,
+    };
+
+    const messageRequestResponse = new MessageRequestResponse(messageRequestResponseParams);
```

Comment:
can you move that `fromHexToArray` to the `MessageRequestResponse` constructor instead? And just pass the string as is